samcli/lib/utils/osutils.py

```diff
@@ -154,3 +155,21 @@ def copytree(source, destination, ignore=None):
             copytree(new_source, new_destination, ignore=ignore)
         else:
             shutil.copy2(new_source, new_destination)
+
+
+def convert_files_to_unix_line_endings(path: str, target_files: Optional[List[str]] = None) -> None:
```

Comment:
There is a test within sam build for java which uses gradlew, which could use this helper function. Right now I believe, it changes the line endings before starting the test, could you change that test to use this helper function as well?

Reply:
Updated.